pkg/chartutil/chartfile.go

```diff
@@ -28,10 +28,10 @@ import (
 	"k8s.io/helm/pkg/proto/hapi/chart"
 )
 
-// ApiVersionV1 is the API version number for version 1.
+// APIVersionV1 is the API version number for version 1.
 //
-// This is ApiVersionV1 instead of APIVersionV1 to match the protobuf-generated name.
-const ApiVersionV1 = "v1"
+// This is APIVersionV1 instead of APIVersionV1 to match the protobuf-generated name.
+const APIVersionV1 = "v1"
```

Comment:
As annoying as it is, we cannot change this one because it is part of the public API. The comment even explains why we did it in the first palce.